contrib/luastatus-9999.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+CMAKE_IN_SOURCE_BUILD=1
+inherit cmake-utils
+
+DESCRIPTION="Universal status bar content generator"
+HOMEPAGE="https://github.com/shdown/luastatus"
+
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	SRC_URI=""
+	EGIT_REPO_URI="git@github.com:shdown/${PN}.git"
+	KEYWORDS="~amd64 ~x86"
+else
+	SRC_URI="https://github.com/shdown/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="amd64 x86"
+fi
+
+BARLIBS="
+	${PN}_barlibs_dwm
+	${PN}_barlibs_i3
+	${PN}_barlibs_lemonbar
+"
+
+PLUGINS="
+	+${PN}_plugins_alsa
+	+${PN}_plugins_fs
+	+${PN}_plugins_mpd
+	+${PN}_plugins_pipe
+	+${PN}_plugins_timer
+	+${PN}_plugins_xkb
+	+${PN}_plugins_xtitle
+"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="${BARLIBS} ${PLUGINS}"
+
+DEPEND=""
+RDEPEND="${DEPEND}
+	dev-lang/lua
+	dev-lang/luajit
```

Comment:
luastatus requires *either* lua *or* luajit. And gentoo has the `luajit` USE flag, so I think you should use it here.